gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -170,7 +173,14 @@ gpii.flowManager.local.setSettings = function () {
     return fluid.promise().resolve();
 };
 
+// Auto save updated preferences received from PSP or QSS
 gpii.flowManager.local.savePreferences = function (that, gpiiKey, preferences) {
```

Comment:
Please add docs comment to this function, especially indicating that preferences is an optional argument and that there is a special meaning to its being missing